docs/pages/server-access/guides/ec2-discovery.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Configure Teleport to Automatically Discover EC2 instances
+description: How to configure Teleport to install itself EC2 instances.
+---
+
+Teleport's SSH Service can be configured to automatically discover, and install
+Teleport on newly provisioned EC2 nodes.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- AWS account with EC2 instances and permissions to create and attach IAM policies as well as permission to create System Manager documents.
+- A running Teleport Node. See the [Server Access Getting Started Guide](../getting-started.mdx) for how to add a Node to your Teleport cluster.
```

Comment:
"Node" typically means "process running the SSH service"

Your new discovery service can run anywhere right? It could run in the Teleport Auth service, for example.